sematic/api/endpoints/tests/test_runs.py

```diff
@@ -22,11 +22,13 @@
 from sematic.calculator import func
 from sematic.db.models.resolution import Resolution, ResolutionStatus
 from sematic.db.models.run import Run
-from sematic.db.queries import get_run, save_resolution, save_run
+from sematic.db.queries import get_run, save_resolution, save_run,\
```

Comment:
Probably want to execute `make fix` and then `make pre-commit`; the linter will complain about the `\` line break.

Reply:
Thanks. 👍  Done as such.